packages/shared/src/hooks/squads/usePostToSquad.ts

```diff
@@ -142,24 +222,42 @@ export const usePostToSquad = ({
   });
 
   const isPosting =
```

Comment:
Jooooh this file is insanity guys 😅
I'm gonna approve it, but it's quite messy

Reply:
Agree, I think it would be better for each posting type to have its own hook